services/configuration/src/lib.rs

```diff
@@ -75,32 +75,42 @@ extern crate exonum_testkit;
 #[cfg(test)]
 #[macro_use]
 extern crate pretty_assertions;
+extern crate serde_json;
+extern crate toml;
 
-pub use errors::ErrorCode;
```

Comment:
This error code used in web api and is a part of public interface, please return back this reexport